tests/test_interrupted_sync.py

```diff
@@ -0,0 +1,201 @@
+                'collections': {
+                    'updated_at': '2022-09-05T14:54:55.000000Z'
+                },
+                'emails': {
+                    'email_sent_timestamp': '2022-09-06T03:00:10.000000Z'
+                },
+                'order_fulfillments': {
+                    '1234567890': '2022-09-02T04:32:15.000000Z',
+                    '1234567890': '2022-09-02T04:32:15.000000Z'
+                },
+                'orders': {
+                    'updated_at': '2022-09-02T14:18:36.000000Z'
+                },
+                'product_reviews': {
+                    '1111111111': '2021-01-01T00:00:00.000000Z',
+                    '2222222222': '2021-01-01T00:00:00.000000Z'
+                },
+                'product_variants': {
+                    '66666666': '2022-08-05T09:02:07.000000Z',
+                    '77777777': '2021-07-15T09:02:07.000000Z',
+                    '88888888': '2021-01-01T00:00:00.000000Z',
+                    'currently_syncing': '77777777'
+                },
+                'reviews': {
+                    'updated_at': '2021-01-01T00:00:00.000000Z'
+                }
+            }
+        }
+        Test Cases:
+        - Verify an interrupted sync can resume based on the currently_syncing and stream level bookmark value.
+        - Verify only records with replication-key values greater than or equal to the stream level bookmark 
+          are replicated on the resuming sync for the interrupted stream.
+        - Verify the pending streams are replicated following the interrupted stream in the resuming sync. 
+          (All pending streams must replicate before streams that were already synced)
+        """
+        
+        start_date = self.get_properties()['start_date'].replace('Z', '.000000Z')
+        expected_streams = self.expected_streams() - {'product_reviews'}
+        expected_replication_keys = self.expected_replication_keys()
+        expected_replication_methods = self.expected_replication_method()
+        LOGGER.info(f"expected_replication_keys = {expected_replication_keys} \n expected_replication_methods = {expected_replication_methods}")
+
+        ##########################################################################
+        # First Sync
+        ##########################################################################
+        conn_id = connections.ensure_connection(self)
+
+        # Run in check mode
+        found_catalogs = self.run_and_verify_check_mode(conn_id)
+
+        # table and field selection
+        catalog_entries = [catalog for catalog in found_catalogs
+                           if catalog.get('tap_stream_id') in expected_streams]
+        
+        self.perform_and_verify_table_and_field_selection(
+            conn_id, catalog_entries)
+
+        # Run a first sync job using orchestrator
+        first_sync_record_count = self.run_and_verify_sync(conn_id)
+        first_sync_records = runner.get_records_from_target_output()
```

Comment:
value for var **first_sync_records** is not being used.. so remove it

Reply:
done